src/Gateway/PayplugGateway.php

```diff
@@ -871,6 +871,7 @@ public function process_standard_payment($order, $amount, $customer_id)
 					$payment_data['allow_save_card'] = false;
 					$payment_data['save_card'] = true;
 					$payment_data['force_3ds'] = true;
+					$payment_data['metadata']['subscription'] = 'subscription';
```

Comment:
this meta data is not being sent

Reply:
<img width="684" alt="image" src="https://github.com/user-attachments/assets/e3bd9cc1-a820-4032-b9fb-e73049b51840" />

Please always provide more details.
What are the product configuration?